modules/email_attestation.js

```diff
@@ -56,29 +72,42 @@ function postAndWriteAttestation(transaction_id, attestor_address, attestation_p
 					}
 
 					db.query(
-						`UPDATE attestation_units 
+						`UPDATE attestation_units
 						SET attestation_unit=?, attestation_date=${db.getNow()}
 						WHERE transaction_id=?`,
 						[unit, transaction_id],
 						() => {
-							let device = require('byteballcore/device.js');
-							let text = "Now your email is attested, see the attestation unit: https://explorer.byteball.org/#"+unit;
-
-							if (src_profile) {
-								let private_profile = {
-									unit: unit,
-									payload_hash: objectHash.getBase64Hash(attestation_payload),
-									src_profile: src_profile
-								};
-								let base64PrivateProfile = Buffer.from(JSON.stringify(private_profile)).toString('base64');
-								text += "\n\nClick here to save the profile in your wallet: [private profile](profile:"+base64PrivateProfile+"). " +
-									"You will be able to use it to access the services that require a proven email address.";
-							}
-
-							text += "\n\n" + texts.weHaveReferralProgram();
-							device.sendMessageToDevice(row.device_address, 'text', text);
-							callback(null, unit);
-							unlock();
+							db.query(
+								`SELECT lang FROM attestation_units as au JOIN attestations as a ON a.unit = au.attestation_unit	JOIN users as u ON a.address = u.user_address WHERE transaction_id = ? LIMIT 1`,
```

Comment:
`attestation_units as au` please keep table names as is unless a new name is clearer https://github.com/byteball/byteballcore/blob/master/styleguide.md#sql

Reply:
fixed